packages/core/.eslintrc.js

```diff
@@ -8,4 +8,7 @@ module.exports = {
       project: [`${__dirname}/test/tsconfig.json`],
     },
   }],
+  rules: {
+    'import/order': 'off',
```

Comment:
@ephys Is it okay to add this rule here? On running `yarn lint-no-fix` the eslintrc.js inside the. packages/core takes precedence hence the errors. And this way we could remove the `// eslint-disable-next-line import/order`? What do you think?

Reply:
I think we prefer to leave this rule on. The instances of disabling the rule in the files were because we are mixing require and import in a few TS files due to our incremental migration but they should be getting fixed slowly.

Reply:
#15591 helps with reducing the disabling of this rule. When you undo the changes to the validator-js test file you should pass the linter. If in the future we would disable this rule or a different rule we should do it in the root .eslintrc file btw, not in just a single package.

Reply:
Thanks!

Reply:
That PR has been merged so you should be able to undo the changes to the .eslintrc file and continue your work